src/binaryen.cpp

```diff
@@ -569,11 +569,15 @@ void BinaryenEngine::verifyContract(wasm::Module & module)
     "Contract export (\"memory\") missing."
   );
 
+/*
+***** in the metering benchmarks, inline-block and inline-super both export the inline useGas functions
```

Comment:
Why do they export it?

Reply:
It was a workaround to make the wast code use the debug name `useGas` for the function (if the inline function wasn't exported, its name would get mangled by the metering injector).